features/lib/fast-selenium.rb

```diff
@@ -0,0 +1,19 @@
+# copy pasta from https://www.browserstack.com/automate/ruby
```

Comment:
Are there any licensing issues around this? It's probably fine, but IANAL

Reply:
Doubtful since it came right out of the docs